qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -272,29 +255,72 @@ def defaults(self, refresh: bool = False) -> Optional[PulseDefaults]:
 
         return self._defaults
 
-    def jobs_limit(self) -> BackendJobsLimit:
-        """Return the jobs limit for the backend.
+    def job_limit(self) -> BackendJobLimit:
+        """Return job limit for the backend.
+
+        The job limit information may include the current number of
+        jobs running on the backend at this time and the maximum number
+        of concurrent jobs a user could submit to the backend at a time.
 
         Note:
-            If `BackendJobsLimit.maximum_jobs = -1`, then there
-            are no limits to the maximum number of jobs that could
-            be submitted on the backend.
+            The job limit information for the backend is given for
+            this account (i.e. a specific backend with a specific
+            provider).
```

Comment:
How about
`The job limit information for the backend is provider specific. For example, if you have access to the same backend via different providers, the job limit information might be different for each provider.
`